Formula/jpeg-turbo.rb

```diff
@@ -30,6 +31,10 @@ def install
     cp Dir["#{Formula["libtool"].opt_share}/libtool/*/config.{guess,sub}"], buildpath
     args = %W[--disable-dependency-tracking --prefix=#{prefix} --with-jpeg8 --mandir=#{man}]
 
+    if build.with? "java"
+      args.push "--with-java"
```

Comment:
`args << "--with-java" if build.with? "java"`

Also, does this require Java to be installed? If so, add a `depends_on :java` above.